# Author : Erik de Castro Lopo <erikd@mega-nerd.com>
# Description : Fix compiler error on big endian systems
 Upstream ships a haskell source file that was generated by the alex lexer
 generator and that file requires the BangPatterns pragma on big endian
 systems. 
# Date : Sun, 22 Apr 2012 13:24:38 +1000

Index: haskell-language-javascript-0.4.10/src/Language/JavaScript/Parser/Lexer.hs
===================================================================
--- haskell-language-javascript-0.4.10.orig/src/Language/JavaScript/Parser/Lexer.hs	2012-03-09 06:56:23.000000000 +1100
+++ haskell-language-javascript-0.4.10/src/Language/JavaScript/Parser/Lexer.hs	2012-04-22 13:03:07.000000000 +1000
@@ -1,4 +1,4 @@
-{-# LANGUAGE CPP,MagicHash #-}
+{-# LANGUAGE CPP,MagicHash,BangPatterns #-}
 {-# LINE 1 "src-dev/Language/JavaScript/Parser/Lexer.x" #-}
 
 
